.travis.yml

```diff
@@ -6,6 +6,8 @@ matrix:
       env: TOXENV=py27,coverage
     - python: 3.6
       env: TOXENV=py36,coverage
+    - python: 3.6
```

Comment:
This makes pylint and flake8 run only for python 3? Wouldn't it be desirable to run the linters for both versions of python?

Reply:
Or would that be redundant?